packages/fxa-content-server/app/scripts/lib/validate.js

```diff
@@ -273,6 +275,16 @@ var Validate = {
 
     return areAllValid;
   },
+
+  /**
+   * Check if the utm param is valid.
+   *
+   * @param {String} value
+   * @returns {Boolean}
+   */
+  isUtmValid(value) {
+    return utmRegex.test(value) && value.length <= 128;
```

Comment:
Instead of specifying length separately, could you replace the `+` with `{0,128}` in the regex?